### ISSUE TITLE
Problem with league/oauth1-client version 1.6.2 upgrading to 1.7

### ISSUE THREAD

**Issue by atiehmokhtari** [thephpleague/oauth1-client#105]
it seems that there is something wrong with upgrading guzzlehttp and the socialite required package league/oauth1-client.
I just tried to upgrade my laravel to laravel 7.0 and also my socialite to 4.4 and guzzlehttp/guzzle package to 7.0 the latest but the package of league/oath1-client downgrades from 1.7 to 1.6.2 which uses guzzle/guzzle instead of guzzlehttp/guzzle. so I am facing the deprecated warning
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.

actually I have deleted composer.lock and vendor folder and then composer update but no help.
and also I had problem and error 500 while I wanted to login through google

**Comment by casperwilkes** [comment]
I'm dealing with the same issue trying to integrate Twitter login functionality through Socialite. Took me a minute, but I tracked the issue down to the same issue, the oauth1-client uses guzzle which is causing composer 2 to go crazy, but is also causing another issue for me which is the same as #103

**Comment by bencorlett** [comment]
This should be fixed in the latest development branch (and the incoming version). I'll close this issue off once testing and the release is complete :)

**Comment by bencorlett** [comment]
This should be fixed in 1.8.0 - please reopen if there's still an issue :)
